packages/camera/camera_platform_interface/test/method_channel/method_channel_camera_test.dart

```diff
@@ -352,8 +375,8 @@ void main() {
     });
 
     group('Function Tests', () {
-      MethodChannelCamera camera;
-      int cameraId;
+      late MethodChannelCamera camera;
+      int? cameraId;
```

Comment:
I think you should make `cameraId` non-nullable since `camera.createCamera` sets it in `setUp`. This way you don't have to null check `cameraId` in every test.

Reply:
Good catch, I have updated this to `late int cameraId;` and removed all the unnecessary null checks.